_field-types/supported-field-types/constant-keyword.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: default
+title: Constant keyword
+nav_order: 71
+has_children: false
+parent: String field types
+grand_parent: Supported field types
+---
+
+# Constant keyword field type
+
+A constant keyword field uses the same value for all documents in the index. 
+
+When a search request spans multiple indexes, you can filter on a constant keyword field to match documents from indexes with the given constant value but not from indexes with a different value.
+
+## Example
+
+The following query creates a mapping with a constant keyword field:
+
+```json
+PUT romcom_movies
+{
+  "mappings" : {
+    "properties" : {
+      "genre" : {
+        "value" : "Romantic comedy"
+      }
```

Comment:
This example does not work. The "type" field is missing. It must look something like this:

` ...
     "genre" : {
        "type": "constant_keyword",
        "value" : "Romantic comedy"
      }
     ...
`